setup.py

```diff
@@ -23,6 +23,7 @@
         "click>=7.0",
         "toml>=0.10.0",
         "pyyaml>=3.0",
+        "awscli==1.20.0",
```

Comment:
Mb let's relax that requirement to something like:
```suggestion
        "awscli>=1.19.0",
```

Reason: if someone wants to install the `NE` package together with other packages, which require `awscli` - fighting with the dependencies might hurt a lot.
Maybe, that also caused errors in CI..
WDYT?